base.php

```diff
@@ -2234,7 +2234,8 @@ function unload($cwd) {
 	*	@return mixed
 	*	@param $key string
 	**/
-	function offsetexists($key) {
+        #[\ReturnTypeWillChange]
```

Comment:
Maybe we should just change the method to do `!== false` check so that boolean is always returned. This would be technically a BC break but hopefully no-one is using `ArrayAccess::offsetExists` to get `ttl`. And if they do, they can trivially replace it with `exists`.

Reply:
That will not actually help since PHP shows the warning based on return type hint, even for child class methods that have no type hints: https://php.watch/versions/8.1/internal-method-return-types#no-return-type

Reply:
Seems there is no way around breaking with php5.4
Or include some circumstances.

Reply:
> 

Composer is a great utility because people can lock in versions of dependencies their application may require via the composer.json file. We shouldn't be worried so much trying to support ancient versions of PHP.